addons/dynGroups/functions/fn_dynamicGroups.sqf

```diff
@@ -209,6 +211,8 @@ switch (_mode) do
 			{
 				["UnregisterGroup: Group (%1) unregistered and deleted (%2)", _group, _keep] call BIS_fnc_logFormat;
 			};
+			
+			["GRAD_dynGroups_unregisterGroup", [_group, _name]] call CBA_fnc_globalEvent;
```

Comment:
hmm group may already be objNull here, right? Could it be better to call the event *before* the unregister is done?